### PR TITLE
fixing unnecessary view in empty main activity

### PR DESCRIPTION
**Pull request by JanBan**

### CODE REVIEW

File: Friendly-plans/app/src/main/java/pg/autyzm/friendly_plans/child_app/view/MainActivity.java [hunk 1]
```diff
@@ -20,23 +20,20 @@
...
+            startActivity(taskList);
             setUpView();
         }
         //Todo - else: redirect to finish activity
     }
 
-    private void setUpView() {
-        setContentView(R.layout.child_app_activity_main);
-    }
+    private void setUpView() { }
```
Comment (SuperJarek) [inline Friendly-plans/app/src/main/java/pg/autyzm/friendly_plans/child_app/view/MainActivity.java]: If it should be removed, it should method setUpView and xml view should be removed as well.
Not sure if should though. I would rahter move setUpView method under if statement. Original app has default view with happy face if there is no plan / plan was done. That's how that view was supposed to work.

Comment (JanBan) [reply Friendly-plans/app/src/main/java/pg/autyzm/friendly_plans/child_app/view/MainActivity.java]: true, done

File: Friendly-plans/app/src/main/java/pg/autyzm/friendly_plans/child_app/view/MainActivity.java [hunk 2]
```diff
@@ -26,13 +26,15 @@ protected void onCreate(Bundle savedInstanceState) {
         ((App) getApplication()).getAppComponent().inject(this);
         activePlan = childPlanRepository.getActivePlan();
         if (activePlan != null) {
-            Intent taskList = new Intent(MainActivity.this, TaskListActivity.class);
-            startActivity(taskList);
-            setUpView();
+            Intent planList = new Intent(MainActivity.this, TaskListActivity.class);
```
Comment (SuperJarek) [inline Friendly-plans/app/src/main/java/pg/autyzm/friendly_plans/child_app/view/MainActivity.java]: taskList was more precise. Stay with that, that was good idea.

Comment (JanBan) [reply Friendly-plans/app/src/main/java/pg/autyzm/friendly_plans/child_app/view/MainActivity.java]: sure, rolled back by mistake, thanks

File: Friendly-plans/app/src/main/java/pg/autyzm/friendly_plans/child_app/view/MainActivity.java [hunk 3]
```diff
@@ -26,8 +26,8 @@ protected void onCreate(Bundle savedInstanceState) {
         ((App) getApplication()).getAppComponent().inject(this);
         activePlan = childPlanRepository.getActivePlan();
         if (activePlan != null) {
-            Intent planList = new Intent(MainActivity.this, TaskListActivity.class);
-            startActivity(planList);
+            Intent taskList = new Intent(MainActivity.this, TaskListActivity.class);
+            startActivity(taskList);
         }
         else
```
Comment (SuperJarek) [inline Friendly-plans/app/src/main/java/pg/autyzm/friendly_plans/child_app/view/MainActivity.java]: unnecessary if

**Review by SuperJarek — APPROVED** [review]

**Pull request merged**